code/datums/diseases/advance/advance.dm

```diff
@@ -0,0 +1,456 @@
+/*
+
+	Advance Disease is a system for Virologist to Engineer their own disease with symptoms that have effects and properties
+	which add onto the overall disease.
+
+	If you need help with creating new symptoms or expanding the advance disease, ask for Giacom on #coderbus.
+
+*/
+
+var/list/archive_diseases = list()
+
+// The order goes from easy to cure to hard to cure.
+var/list/advance_cures = 	list(
+									"sodiumchloride", "sugar", "orangejuice",
+									"spaceacillin", "salglu_solution", "ethanol",
+									"teporone", "diphenhydramine", "lipolicide",
+									"silver", "gold"
```

Comment:
Given that silver and gold are readily produced in any chem dispenser, perhaps those should be changed.